BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/single_file/single_file.rb

```diff
@@ -36,7 +36,12 @@ def self.valid_file_classes #:nodoc:
   end
 
   def pretty_type #:nodoc:
-    self.class.pretty_type + (name =~ /(\.gz|\.Z|\.gz2)$/i ? " (compressed)" : "")
+    self.class.pretty_type + (compressed? ? " (compressed)" : "")
+  end
+
+  def compressed? #:nodoc:
```

Comment:
I think it's Compress/Uncompress at all times now... I never liked that but I've been overruled.